wiki/content/graphql/admin/index.md

```diff
@@ -1,458 +0,0 @@
-+++
-title = "Admin"
```

Comment:
Why are we removing this?

Reply:
I think all docs moved to dgraph-docs. I don't know why this file remains.